Modules/xSharePoint/DSCResources/MSFT_xSPSubscriptionSettingsService/MSFT_xSPSubscriptionSettingsService.psm1

```diff
@@ -0,0 +1,106 @@
+function Get-TargetResource
```

Comment:
Can we name this one MSFT_xSPSubscriptionSettingsServiceApp (add App on the end - keeps the naming consistent with our other service apps)